### ISSUE TITLE
html tags supported?

### ISSUE THREAD

**Issue by andysay** [orlyyani/read-more#2]
i use html tags, this solution will render text with html tags? thanks

**Comment by orlyyani** [comment]
yes, it does support html tags and will not render any text with html tags.

**Comment by akhoury** [comment]
not when using the directive though, you'd have to replace all the `textContent` with `innerHTML` here https://github.com/orlyyani/read-more/blob/6b27a96eeb456bbb4a2d25a143d089449ebe8844/src/ReadMoreDirective.js
